contrib/coraza-spoa.service

```diff
@@ -45,7 +45,7 @@ InaccessiblePaths=-/opt
 InaccessiblePaths=-/srv
 #InaccessiblePaths=-/bin
 InaccessiblePaths=-/bin/bash
-inaccessiblepaths=-/bin/find
+InaccessiblePaths=-/bin/find
```

Comment:
Nice catch :smile:

Reply:
I choose to fix this in #133 since it will be more fitting there.